cloud-controller-manager/cmd/digitalocean-webhook-server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM golang:1.19 as builder
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+COPY ../../../../vendor/ vendor/
```

Comment:
The relative path means that we need to be very careful about where the docker image is being built from when you do docker build. vendor/ is in the same level as go.mod, go.sum. Why was the relative path even added here, but COPY works without it for go.mod?